### PR TITLE
Update condition when call manifest failed

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by qiujian16** [comment]
fixes https://github.com/open-cluster-management-io/addon-framework/issues/98

File: pkg/addonmanager/constants/constants.go [hunk 1]
```diff
@@ -14,6 +14,25 @@ const (
 
 	// PreDeleteHookFinalizer is the finalizer for an addon which has deployed hook objects
 	PreDeleteHookFinalizer = "cluster.open-cluster-management.io/addon-pre-delete"
+
+	// AddonManifestApplied is a condition type representing whether the manifest of an addon
+	// is applied correctly.
+	AddonManifestApplied = "ManifestApplied"
+
+	// AddonManifestAppliedReasonWorkApplyFailed is the reason of condition AddonManifestApplied indicating
+	// the failuer of apply manifestwork of the manifests
```
Comment (zhiweiyin318) [inline pkg/addonmanager/constants/constants.go]: typo: failuer

**Comment by zhujian7** [comment]
@qiujian16 The e2e failed, please take a look

**Comment by qiujian16** [comment]
e2e is fixed.

**Pull request merged**